service-runner/src/main/resources/config/assets.yaml

```diff
@@ -208,15 +237,15 @@ assets:
       buy_delivery_methods:
         - name: WIRE
           description: Have USD sent directly to your bank account.
-    sep38_enabled: true
   - schema: iso4217
     code: CAD
     significant_decimals: 7
     sep31:
```

Comment:
```suggestion
    sep31:
      enabled: true
```

Reply:
Is this how we interpret if service is enabled now? 
I can use some clarification so that I can clean up the test asset config all at once.

This PR assume service is **NOT** enabled unless `enabled` is set to true

Reply:
I just tested it and my assumption is right.